Makefile

```diff
@@ -10,14 +10,16 @@ GO_BUILD_ARGS=CGO_ENABLED=0 GO111MODULE=on
 # TODO (suhanime): Pin go versions and lint
 # TODO (suhanime): Enable linting when we don't have unimplemented methods
 
-# The golang 1.13 image used in CI enforces vendoring. Workaround that by unsetting it.
-ifeq ($(GOFLAGS), -mod=vendor)
-	unexport GOFLAGS
+# Set the go mod vendor flags, if it is not set
+GOFLAGS? = $(shell go env GOFLAGS)
+ifneq "$(findstring -mod=vendor,$(GOFLAGS))" "-mod=vendor"
+GOFLAGS ?= -mod=vendor
```

Comment:
not sure we need this with 1.14+ -- it's now set by default (https://golang.org/doc/go1.14#go-command)